tests/commonUtils.sh

```diff
@@ -15,8 +15,8 @@
 #   limitations under the License.
 
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
-RUNTIME_IMAGE='docker.io/istio/runtime:2017-03-22-21.21.48'
-
+MANAGER_HUB_TAG='docker.io/istio,2017-03-22-21.21.48'
```

Comment:
@frankbu I am confused. Did not we agree yesterday to wait for @ZackButcher  and @kyessenov to update istio/istio with working version of manager and mixer ? I feel that merging the PR will break stuff even more.

@rshriram For integration testing (ie a PR created on istio/istio) we use the version hard coded in the yaml files.